libs/components/popovers/src/lib/modules/dropdown/dropdown-item.component.scss

```diff
@@ -56,7 +56,10 @@
 
     &.sky-dropdown-item-active,
     &:hover {
-      background-color: transparent;
+      a,
+      button {
+        background-color: transparent;
+      }
```

Comment:
The default theme styles were affected in #1308 but the modern styles were not adjusted.